master/docs/relnotes/index.rst

```diff
@@ -42,6 +46,11 @@ Features
 * The Buildbot slave now includes the number of CPUs in the information it supplies to the master on connection.
   This value is autodetected, but can be overridden with the ``--numcpus`` argument to ``buildslave create-slave``.
 
+* The :class:`DockerLatentBuildSlave` image attribute is now renderable (can take properties in account).
```

Comment:
This is cool, and should be mentioned somewhere in the configuration section of the manual